src/Microsoft.OData.Core/Microsoft.OData.Core.txt

```diff
@@ -687,6 +687,7 @@ UriQueryExpressionParser_WithExpected='with' expected at position {0} in '{1}'.
 UriQueryExpressionParser_UnrecognizedWithMethod=Unrecognized with '{0}' at '{1}' in '{2}'.
 UriQueryExpressionParser_PropertyPathExpected=Expression expected at position {0} in '{1}'.
 UriQueryExpressionParser_KeywordOrIdentifierExpected='{0}' expected at position {1} in '{2}'.
+UriQueryExpressionParser_InnerMostExpandRequireFilter=inner most expand require filter() at position {0} in '{1}'.

```

Comment:
>inner most expand require  [](start = 54, length = 26)

(minor editorial) should start with a capital letter. Also, wording suggestion:
 The inner most expand transformation requires a filter transformation at position {0} in '{1}'.